src/components/SuggestedItem/SuggestedItem.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./SuggestedItem.css";
+
+export default props => {
+  const {
+    // flag,
+    handleClick,
+    image,
+    name,
+    pack,
+    index
+  } = props;
+  const toBePacked = pack ? "item--font--packed" : "item--font--unpacked";
+  const borderColor = pack ? " item--border--packed " : " item--border--unpacked ";
+  return (
+    <div
+      className={toBePacked + borderColor +"  item p-0 m-1 rounded"}
+      onClick={handleClick("bundle", index)}
```

Comment:
Similar comments here with both the `handleClick(...)` bug and this being an incorrect element kind (`div` vs `button` or `input`).

Reply:
I added a button element and attacked the handler to it. Definitely makes sense that it should be on a button or input especially for accessibility. Thanks!